sysconfig/jureca-booster/modules.yaml

```diff
@@ -26,17 +26,18 @@ modules:
       suffixes:
           '+profile': 'profile'
           '^coreneuron+knl': 'knl'
+          '~mpi~binary': 'serial'
       filter:
         environment_blacklist: ['CPATH', 'LIBRARY_PATH']
     naming_scheme: '${PACKAGE}/${VERSION}'
     hash_length: 0
     whitelist:
       - 'neurodamus'
+      - 'neurodamus-hippocampus'
+      - 'neurodamus-neocortex'
       - 'neuron'
       - 'py-bluepyopt'
-      - 'python'
-      - 'py-matplotlib'
-      - 'py-ipyparallel'
+      - 'py-bluepy'
```

Comment:
could you keep `'py-ipyparallel'`? I remember someone complained because of PYTHONPATH for ipyparallel was missing.

Edit : I believe this is not required because we are loading default python and ipyparallel from there?

Reply:
I think we removed it because it was not required as you mention.